java/core/src/java/org/apache/orc/impl/RecordReaderImpl.java

```diff
@@ -1249,7 +1249,7 @@ public boolean[] pickRowGroups(StripeInformation stripe,
      * Get the count of exceptions for testing.
      * @return
      */
-    long[] getExceptionCount() {
+    public long[] getExceptionCount() {
```

Comment:
I'm curious why we need to change this. According to the function description, this is used in the test code already, isn't it?
> Get the count of exceptions for testing.

Reply:
Unit test that uses this method, but the test file is not in the same package as RecordReaderImpl.java